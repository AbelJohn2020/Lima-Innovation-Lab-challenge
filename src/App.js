import React from 'react';
import { BrowserRouter as Router, Switch, Route } from "react-router-dom";
import Posts from './components/post/Posts';
import './app.css';
import Header from './components/header/Header';
import CommentItem from './components/comments/CommentItem';
import UserProfile from './components/users/UserProfile';

const App = () => {
    return (
      <Router>
        <div className="AppContainer">
        <Switch>
          <Route exact path="/">
            <Header />
            <Posts />
          </Route>
          <Route exact path="/post/:postId">
            <CommentItem />
          </Route>
          <Route exact path="/user/:userId">
            <UserProfile />
          </Route>
        </Switch>
        </div>
      </Router>
    );
};

export default App;