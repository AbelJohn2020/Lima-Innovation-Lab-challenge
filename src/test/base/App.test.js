import { shallow } from 'enzyme';
import App from '../../App';

describe('Test in <App />', () => {
    test('should return the correct component', () => {
        const wrapper = shallow( <App />);
        expect(wrapper).toMatchSnapshot();
    });
});