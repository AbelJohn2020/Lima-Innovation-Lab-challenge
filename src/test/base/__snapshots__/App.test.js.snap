// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test in <App /> should return the correct component 1`] = `
<div
  className="AppContainer"
>
  <Header />
  <Users />
  <Posts />
</div>
`;
